pkg/policy/identity.go

```diff
@@ -41,11 +46,28 @@ const (
 	// InvalidIdentity is the identity assigned if the identity is invalid
 	// or not determined yet
 	InvalidIdentity = NumericIdentity(0)
+
+	// DefaultAddressSpace is the address space used if none is provided.
+	DefaultAddressSpace = "default"
 )
 
 var (
-	// IdentitiesPath is the to where identities are stored
+	// IdentitiesPath is the path to where identities are stored in the key-value
+	// store.
 	IdentitiesPath = path.Join(kvstore.BaseKeyPrefix, "state", "identities", "v1")
+
+	// IPIdentitiesPath is the path to where endpoint IPs are stored in the key-value
+	//store.
+	IPIdentitiesPath = path.Join(kvstore.BaseKeyPrefix, "state", "endpointIPs", "v1")
+
+	// IPIdentityCache caches the mapping of endpoint IPs to their corresponding
+	// security identities across the entire cluster in which this instance of
+	// Cilium is running.
+	IPIdentityCache = NewIPCache()
```

Comment:
Pass in the cache, which is defined in https://github.com/cilium/cilium/pull/2879:
```
IPIdentityCache = NewIPCache(envoy.NetworkPolicyHostsCache)
```